Makefile

```diff
@@ -2,14 +2,12 @@
 
 auto: build38
 
-build27: PYTHON_VER = python2.7
-build35: PYTHON_VER = python3.5
 build36: PYTHON_VER = python3.6
 build37: PYTHON_VER = python3.7
 build38: PYTHON_VER = python3.8
 build39: PYTHON_VER = python3.9
 
-build27 build35 build36 build37 build38 build39: clean
+build36 build37 build38 build39: clean
 	virtualenv venv --python=$(PYTHON_VER)
```

Comment:
We can actually dump `virtualenv` since we are Python 3+ only now. Instead, we can use the built in module to create a virtual env: `$(PYTHON_VER) -m venv env`

Reply:
It's been a while since I've used make directly myself, mostly it's precommit and tox, heh

I agree with this change, will do so.